<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSAutomation</class>
 <widget class="QDialog" name="SSAutomation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="Combo_Sites">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>351</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_Orders">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>351</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
